Pax.cs

```diff
@@ -352,10 +353,11 @@ private static void RegisterHandlers()
     //Cleanup
     private static void shutdown (object sender, ConsoleCancelEventArgs args)
     {
-      for (int idx = 0; idx < PaxConfig.no_interfaces; idx++)
+      foreach (var device in PaxConfig.deviceMap)
       {
-        PaxConfig.deviceMap[idx].Close();
-      }
+        device.StopCaptureTimeout = TimeSpan.FromSeconds(1);
```

Comment:
It would be useful if there was a comment saying what this line did, and if it was necessary, and if the constant `1` was special, and if the user should be allowed to change this value through a configuration parameter.